src/public/parts/shells/chat/public/src/templates/group/settings/channel_permissions_panel.html

```
@@ -16,42 +16,33 @@ <h2 class="card-title" data-i18n="chat.group.settings.page.tabs.channelPermissio
-							<div class="settings-role-perms">
-								${panel.permRows.map(row => `
-								<div class="flex items-center justify-between gap-2 py-1">
-									<span class="text-sm" data-i18n="chat.group.settings.page.perm.${row.perm}"></span>
-									<div class="join join-horizontal" role="group" data-role-id="${escapeHtml(panel.roleId)}" data-perm="${row.perm}">
-										<button type="button" class="btn btn-xs join-item ${row.state === 'neutral' ? 'btn-active' : 'btn-ghost'}" data-action="channel-perm-state" data-state="neutral" data-i18n="chat.group.settings.page.channelPerms.stateNeutral"></button>
-										<button type="button" class="btn btn-xs join-item ${row.state === 'allow' ? 'btn-success btn-active' : 'btn-ghost'}" data-action="channel-perm-state" data-state="allow" data-i18n="chat.group.settings.page.channelPerms.stateAllow"></button>
-										<button type="button" class="btn btn-xs join-item ${row.state === 'deny' ? 'btn-error btn-active' : 'btn-ghost'}" data-action="channel-perm-state" data-state="deny" data-i18n="chat.group.settings.page.channelPerms.stateDeny"></button>
-									</div>
+			<div class="flex-1 min-w-0 space-y-3">
+				${rolePanels.map(panel => `
+				<details class="settings-role" data-role-panel="${escapeHtml(panel.roleId)}" ${panel.open ? 'open' : ''}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)